torch/_inductor/codegen/wrapper.py

```diff
@@ -713,6 +713,10 @@ def generate(self, is_inference):
             self.write_triton_header_once()
         result = IndentedBuffer()
         result.splice(self.header)
+        # We do not want the cpp header for intermediate const graph. Headers would be
+        # rendered by the main module instead.
+        if V.graph.aot_mode and V.graph.cpp_wrapper and V.graph.is_const_graph:
```

Comment:
Nit: no need to check `V.graph.cpp_wrapper` here because it's `V.graph.aot_mode`.